src/Compilers/CSharp/Test/Semantic/Semantics/PatternMatchingTests2.cs

```diff
@@ -1452,429 +1452,5 @@ class Cat {}
                 );
             var comp = CompileAndVerify(compilation, expectedOutput: @"Fox Cat");
         }
-
```

Comment:
Were these tests removed intentionally?

Reply:
No. Looks like a merge gone bad. Will reinsert them.

---
In reply to: [181198412](https://github.com/dotnet/roslyn/pull/26088#discussion_r181198412) [](ancestors = 181198412)